Pod/Classes/BugShaker.swift

```diff
@@ -8,12 +8,16 @@
 import UIKit
 import MessageUI
 
+public protocol BugShakerDelegate {
+  func ShouldPresentReportPrompt() -> Bool
```

Comment:
The general convention for Swift method names is camelCase. Could we change this to `shouldPresentReportPrompt()`?